bin/main.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+puts "Hello welcome to the Tic Tac Toe Game"
+puts "What is your name? (player X)"
+player_x = gets.chomp
+puts "What is your name? (player O)"
+player_o = gets.chomp
+puts "--------------------------------------"
+puts "This our board"
+puts "_1_|_2_|_3_"
+puts "_4_|_5_|_6_"
+puts " 7 | 8 | 9 "
+
+there_turns = 9
+
+while there_turns > 0
+
+  puts "Pick a number to set the turn 'X' "
```

Comment:
:recycle:  `[REQUIRED]` game tells which player turn it is

Could we at least add a comment to say this statement will change from `X` to `O` for the next player turn? :thinking:

Or something like this:
```suggestion
  player_turn = (there_turns % 2).zero? ? 'O' : 'X'
  puts "Pick a number player #{player_turn}"
```

or
```suggestion
  player_turn = (there_turns % 2).zero? ? player_o : player_x
  puts "Pick a number #{player_turn}"
```

It can also be a comment as I said earlier, it doesn't have to work. The point is to explain know how the game flow will work.